models/index.js

```diff
@@ -0,0 +1 @@
+exports.fetchTopics = require('../models/app.models').fetchTopics
```

Comment:
Not sure why these need to be exported into a different file - _unless_ you're planning on multiple model files (which would be good) and want to require them into controller via one file?